smarts/core/traffic_history_provider.py

```diff
@@ -90,10 +90,15 @@ class _HistoryRow(NamedTuple):
     def _decode_vehicle_type(self, vehicle_type):
         # Options from NGSIM and INTERACTION currently include:
         #  1=motorcycle, 2=auto, 3=truck, 4=pedestrian/bicycle
-        # But we don't yet have glb models for 1 and 4.
```

Comment:
I just went ahead and used the new types in our TrafficHistoryProvider since that's what spurred the issue to be created.